chain/jsonrpc/jsonrpc-tests/res/genesis_config.json

```diff
@@ -75,4 +75,4 @@
   "num_chunk_validator_seats": 300,
   "chunk_producer_assignment_changes_limit": 5,
   "records": []
-}
+}
```

Comment:
looks like an accident

Reply:
This was auto-generated, so I prefer keeping it as is, though it looks no-op.